src/pages/registerUser.vue

```diff
@@ -76,17 +76,22 @@
           <v-col v-if="sucessModal" cols="12">
             <sucessfullSignUp :content="'You have successfully registered'" @closeModals="handleCloseModal" />
           </v-col>
+          <v-col cols="12" class="mt-1 text-center">
+           <socialIcon/>
```

Comment:
not socialIcon 
this component is used to signIn or singup using social handlers 
so name should be according to that @ShivanshSinghnnovative